### DISCUSSION TITLE
What's wrong with my source string? (ICU MessageFormat syntax warning)

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#7041] [Q&A] [answered]
Hello,

I just added the 'final' component for our project. There is [one source string](https://hosted.weblate.org/translate/antennapod/website-documentation/en/?checksum=dfa5af6fe3484189) that has a warning I don't understand:
![afbeelding](https://user-images.githubusercontent.com/11229646/147892400-19a45fbb-ccf9-4f42-bd95-bf8dc04c54bc.png)

The [documentation](https://docs.weblate.org/en/weblate-4.10.1/user/checks.html#check-icu-message-format-syntax) which is linked to, doesn't help me much further either, even after reading about the [ICU message format](https://docs.weblate.org/en/weblate-4.10.1/user/checks.html#icu-messageformat) I have no idea what it is.

This string is just a text from our website (Jekyll, with in this string a liquid tag). Can anyone explain what might be 'wrong' with my string (I'm sure I can just safely dismiss the warning, but I'd like to understand).

**Comment by nijel** [comment, marked as answer]
This is a bug in Weblate, please follow up at https://github.com/WeblateOrg/weblate/issues/7048

**Reply by keunes** [reply]
Awesome, thanks for the quick fix!
